documentation/Configuration.md

```diff
@@ -418,6 +418,7 @@ The following properties are used to configure layout rules in StyleCop Analyzer
 | --- | --- | --- | --- |
 | `newlineAtEndOfFile` | `"allow"` | 1.0.0 | Specifies the handling for newline characters which appear at the end of a file |
 | `allowConsecutiveUsings` | `true` | 1.1.0 | Specifies if SA1519 will allow consecutive using statements without braces |
+| `allowDoWhileOnClosingBrace` | `false` | >1.2.0 | Specifies if SA1500 will allow the `while` expression of a `do-while` loop to be on the same line as the closing brace, as is generated by the default code snippet of Visual Studio |
```

Comment:
I see that there is a 1.2.0-beta in the works:
![image](https://user-images.githubusercontent.com/22459191/92319051-48f98b00-efc9-11ea-93c2-ca44d341c474.png)
So I put the version number here as `>1.2.0` because I don't know how to predict what version this might end up in.

We can change it to something else, or omit these changes from this PR and make another one or something later.